deploy/eck-beats/values.yaml

```diff
@@ -61,9 +61,10 @@ spec:
 
   # Daemonset, or Deployment specification for the type of Beat specified.
   # At least one is required of [daemonSet, deployment].
+  # No default is currently set until https://github.com/helm/helm/pull/11440 is merged.
   # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-beat-configuration.html#k8s-beat-chose-the-deployment-model
   #
-  daemonSet: {}
+  # daemonSet: {}
   # deployment: {}
```

Comment:
It's not from your PR, but we are not concistent with the daemonSet/deployment defaults for Beats and Agent.

Reply:
I believe I made them look consistent in the values @thbkrkr lmk.